tests/neo-vm.Tests/UtVMJson.cs

```diff
@@ -57,7 +60,14 @@ private void TestJson(string path)
                     //File.WriteAllText(realFile, ut.ToJson().Replace("\r\n", "\n"), Encoding.UTF8);
                 }
 
-                ExecuteTest(ut);
+                try
+                {
+                    ExecuteTest(ut);
```

Comment:
New exceptions being launched now @shargon ?

Reply:
Yes, because was hard to detect for me which file was wrong.